cloud_utils/slack_utils.py

```diff
@@ -0,0 +1,38 @@
+import traceback
+from typing import Dict, Optional, Text
+
+import gamla
+import requests
+
+
+def send_message_to_webhook(webhook_url: Text, text: Text, data: Optional[Dict]):
```

Comment:
data should not be optional imo, it's better to force user to send it.

Reply:
why? most of the times we want to only post a text message to slack, without attachments or blocks (the data)

Reply:
actually i changed the implementation, ptal

Reply:
lgtm